### DISCUSSION TITLE
On network error trying to fetch 4 times, configurable?

### DISCUSSION THREAD

**Discussion by devidw** [octokit/octokit.js#2351] [Q&A] [answered]
When I force network errors via DevTools request blocking 4 requests are blocked after each other. However, the application only makes one request.

![Screenshot 2022-11-12 at 17 18 45](https://user-images.githubusercontent.com/68775926/201483779-2a60cf11-2497-4fa0-859c-f8dd40205e01.png)

![Screenshot 2022-11-12 at 17 18 55](https://user-images.githubusercontent.com/68775926/201483784-3350d63b-6af9-4a37-b27f-2c0b4e113e51.png)

It seems Octokit tries to fetch 4 times in a row, when there is a network error.

I could not find any resources on this, if it's true, can it be configured to stop after one failed request?

Thanks 🙏

**Comment by gr2m** [comment, marked as answer]
Yes, you can disable request retries like this:

```js
new Octokit({ retry: { enabled: false } })
```

The `retry` option is current not documented in this repository's README. If you could send a pull request and put it below the `throttle` option, that would be great. You could amend the `throttle` description that it can be disabled by setting it to `{ enabled: false }` as well
